### PR TITLE
fixes for decision table on a package

### PR DESCRIPTION
**Pull request by alfretoine**
## Self proof reading checklist

- [ ] Did I run Pronto and fixed all warnings
- [ ] Make sure all naming and code will remain understandable in 6 month by someone new to the project or by you.
- [ ] Did I test the right thing?
- [ ] Did I test corner cases, non happy path (defensive testing)?
- [ ] Check that I used the ad-hoc patterns and created files accordingly (Presenters, Services, Search object, Form object,...)
- [ ] Check code efficiency with record intensive project
- [ ] Update documentation,readme accordingly

Thanks!

### CODE REVIEW

File: spec/models/rules_spec.rb [hunk 1]
```diff
@@ -231,6 +284,12 @@
       p
     end
 
+    let(:quality_package) do
```
Comment (mestachs) [inline spec/models/rules_spec.rb]: is this still needed ? ( describe "Package involving a decision table" )
